translations/pt-br/04-ferramentas-utilizadas.md

```diff
@@ -0,0 +1,142 @@
+# Ferramentas utilizadas
+
+Para ter sucesso na sua jornada de código aberto, é essencial familiarizar-se com as ferramentas e plataformas comumente utilizadas pela comunidade. Essas ferramentas ajudarão você a colaborar de forma eficaz, gerenciar seus projetos e contribuir para projetos de código aberto. Neste capítulo, discutiremos as seguintes ferramentas essenciais para colaboração de código aberto:
+
+- GitHub & Git
+- CodeSee
+- VS Code (O editor)
+- Discord
+
+## Pré-requisitos
+
+Antes de completar o seguinte passo a passo, complete o seguinte:
+- Crie uma [conta no GitHub](https://github.com/)
+- Baixe o [VS Code](https://code.visualstudio.com/)
+
+## GitHub & Git
+
+### Introdução ao Git
+
+Git é um sistema de controlo de versão distribuído projetado para lidar com projetos pequenos e muito grandes com velocidade e eficiência. Foi criado por Linus Torvalds, o criador do kernel Linux, para gerir o desenvolvimento do próprio kernel Linux.
+
+Git permite que os desenvolvedores acompanhem as alterações no seu código, colaborem com outros desenvolvedores e gerenciem várias versões dos seus projetos. Alguns dos recursos principais do Git incluem:
+
+- **Branch**: Git permite que os desenvolvedores criem ramificações, cópias separadas da sua base de código. Isso permite que eles trabalhem em novos recursos ou correções de bugs sem afetar a base de código principal. Uma vez que as alterações estejam concluídas, eles podem mesclar a ramificação de volta para a base de código principal.
+- **Área de preparação(Stagging area)**: Git fornece uma área de preparação onde os desenvolvedores podem preparar as suas alterações antes de as comprometerem com o repositório. Isso permite que eles revisem e organizem as suas alterações antes de as tornarem permanentes.
```

Comment:
```suggestion
- **Ambiente de preparação(Stagging area)**: Git fornece uma área de preparação onde os desenvolvedores podem preparar as suas alterações antes de as aplicarem ao repositório. Isso permite que eles revisem e organizem as suas alterações antes de as tornarem permanentes.
```
"Ambiente" fits better as is widely used "Ambiente de Produção", "Ambiente de Homologação", "Ambiente de Teste"... https://pt.wikipedia.org/wiki/Ambiente_de_implanta%C3%A7%C3%A3o

I'm unsure if "commit" should be translated. Either way, "aplicar" sounds better than "comprometer" IMO